README.md

```diff
@@ -98,3 +99,23 @@ Application Options:
   -r, --run-once           Check and install new software and exit
   -s, --output-systemd     Enable output to systemd
 ```
+
+
+Authentication
+--------------
+
+As described on the [hawkBit Authentication page](https://www.eclipse.org/hawkbit/concepts/authentication/)
+in the "DDI API Authentication Modes" section, a device can be authenticated
+with a security token. A security token can be either a "Target" token or a
+"Gateway" token. The "Target" security token is specific to a single target
+defined in hawkBit. In the RAUC hawkBit updater's configuration file it's
+referred to as "auth_token". The "Gateway" token allows all targets on the
```

Comment:
Just noted that this is misleading. It is meant to be used for a *Gateway* that handles (and possibly authenticates) multiple targets, it is not primarily meant for directly authenticating targets!

Reply:
> Just noted that this is misleading. It is meant to be used for a _Gateway_ that handles (and possibly authenticates) multiple targets, it is not primarily meant for directly authenticating targets!

Reworded to match the hawkBit documentation.

Reply:
@serg-z Thank you for adapting the documentation!